### PR TITLE
Post component

### PR DESCRIPTION
**Pull request by AkshatPandey1**
# Features
- Post component 
- CreatePost component 

# Todo 
- Comments 
- Share 
- Endpoint connections

### CODE REVIEW

**Comment by Mattwmaster58** [comment]
glad we don't have to support HTML :)

File: api/user/author/authors.py [hunk 1]
```diff
@@ -45,6 +45,13 @@ def get_single_author(author_id: str):
     return found_author.getJSON()
 
 
+@authors_bp.route("/authenticated_user_id", methods=["GET"])
+@login_required
+def manpreet():
+    if current_user.is_authenticated:
+        return {"id": current_user.id}
+
+
```
Comment (Mattwmaster58) [inline api/user/author/authors.py:54]: is this still in use? haven't found any reference

**Review by Mattwmaster58 — CHANGES_REQUESTED** [review]

File: api/user/author/authors.py (hunk 1 above)
Comment (AkshatPandey1) [reply api/user/author/authors.py:54]: Gonna be using it in the future to get the user id of the user that has logged in

**Comment by Mattwmaster58** [comment]
I would say hook up to a testing backend endpoint for now - we should be able to easily swap out for the real user endpoint easily then. What do yo uthink?

**Comment by AkshatPandey1** [comment]
> I would say hook up to a testing backend endpoint for now - we should be able to easily swap out for the real user endpoint easily then. What do yo uthink?

Seems like a good idea, but just curious, why do we need to hook up to a testing backend endpoint first?

**Comment by Mattwmaster58** [comment]
In my mind it will be easy to swap out one endpoint for another and help iron any bugs that might occur there than swap from "frontend mock data" to backend endpoint but this is all just a guess on my part. If you think it'll be no trouble going straight shot to the "real" backend endpoint let's leave it the way it is now

File: api/user/author/authors.py (hunk 1 above)
Comment (Mattwmaster58) [reply api/user/author/authors.py:54]: I guess we do need that to get their feed...:facepalm:

**Comment by AkshatPandey1** [comment]
Sounds good, we can go test with them first. I'll put it in the endpoints branch I'll make once this branch is merged with master.

**Comment by Mattwmaster58** [comment]
superseded by #77

**Pull request closed without merging**